### ISSUE TITLE
Not feedback when elem is null in the browser

### ISSUE THREAD

**Issue by criloz** [Raynos/mercury#79]
when in a web browser the `elem` is still not loaded [this line](https://github.com/Raynos/mercury/blob/e480a8aa0f758f8bf4657c8375333c0d8031bcc5/index.js#L54) occults that error  and don't give any type of feedback

this can be painful for newbies

**Comment by kumavis** [comment]
+1

**Comment by klemola** [comment]
Should Mercury execution continue if the element does not exist?

The current `if (elem)` check could be changed to something like below:

```
if (!elem) {
    throw new ReferenceError('Element does not exist. Mercury cannot be initialized.');
}
```

**Comment by kumavis** [comment]
sounds good to me. @Raynos thoughts?

**Comment by Raynos** [comment]
:+1:

**Comment by klemola** [comment]
Great! I can make a pull request later.

**Comment by zakm** [comment]
I think the change above breaks the server-side rendering example.

**Comment by kumavis** [comment]
can someone confirm / fix ?

**Comment by nlocnila** [comment]
server-side rendering examples breaks on moving to `virtual-dom@2.1.1` and the above change does break the server-side rendering example

**Comment by matthewp** [comment]
Yes, this does indeed break server-side rendering.

**Comment by matthewp** [comment]
@kumavis What if we allowed you to pass `true` instead of `null`, that would preserve this fix but still allow for SSR.

Instead of:

```js
hg.app(null, app, App.render, {
    initialTree: prevTree,
    target: targetElem
});
```

We do:

```js
hg.app(true, app, App.render, {
    initialTree: prevTree,
    target: targetElem
});
```

Would you accept this as a fix @kumavis ?

**Comment by matthewp** [comment]
@Raynos @kumavis I've submitted a fix in #208.
